graphql/introspection/schema.go

```diff
@@ -11,12 +12,20 @@ type Schema struct {
 }
 
 func (s *Schema) Types() []Type {
-	types := make([]Type, 0, len(s.schema.Types))
+	typIndex := map[string]Type{}
```

Comment:
Is the spelling here intentional? 
```suggestion
	typeIndex := map[string]Type{}
```

Reply:
Nope, fixed!